### PR TITLE
chap7

### PR DESCRIPTION
**Pull request by matTroiCuaHieu**
image

### CODE REVIEW

**Comment by matTroiCuaHieu** [comment]
ready

**Comment by matTroiCuaHieu** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = t "shared.welcome_to_the_sample_app"
+      redirect_to @user
+    else
+      flash[:danger] = t "shared.error_signup"
+      render "new"
```
Comment (ngocvt-0484) [inline app/controllers/users_controller.rb:17]: render :new

**Comment by matTroiCuaHieu** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**